src/Imageflow.Server.Storage.RemoteReader/RemoteReaderServiceExtensions.cs

```diff
@@ -1,21 +1,41 @@
 ﻿using Imazen.Common.Storage;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
+using System;
+using System.Net.Http;
 
 namespace Imageflow.Server.Storage.RemoteReader
 {
     public static class RemoteReaderServiceExtensions
     {
-        public static IServiceCollection AddImageflowRemoteReaderService(this IServiceCollection services,
-    RemoteReaderServiceOptions options)
+        public static IServiceCollection AddImageflowRemoteReaderService(this IServiceCollection services
+            , RemoteReaderServiceOptions options
+            )
         {
             services.AddSingleton<IBlobProvider>((container) =>
             {
                 var logger = container.GetRequiredService<ILogger<RemoteReaderService>>();
-                return new RemoteReaderService(options, logger);
+                var http = container.GetRequiredService<IHttpClientFactory>();
```

Comment:
Is there a default IHttpClientFactory registered? If so this means configuration can be a lot simpler than I thought.

Reply:
There isn't a factory registered by default, but this is all that is required:
```
services.AddHttpClient();
```
Would you like me to update the sample to use the simplest config, and comment the other options so they are there for reference?